packages/webview_flutter/webview_flutter_android/lib/src/android_webview.dart

```diff
@@ -627,7 +638,7 @@ class WebResourceRequest {
   final String method;
 
   /// Gets the headers associated with the request.
-  final Map<String, String> requestHeaders;
+  final Map<String, String>? requestHeaders;
```

Comment:
Collections should generally be non-nullable, otherwise there are two ways of representing the same thing (null and empty). Callers can use `??` to pass an empty collection when they have null.

Reply:
That's a good point. I changed it back to being non-null.